client/components/LearnerViewDirective.js

```diff
@@ -296,9 +296,9 @@ tie.directive('learnerView', [function() {
 
 
         $scope.questionIds = questionSet.getQuestionIds();
-        $scope.unlockedQuestions = [];
+        $scope.completedQuestions = [];
         for (var i = 0; i < $scope.questionIds.length; i++) {
-          $scope.unlockedQuestions.push(false);
+          $scope.completedQuestions.push(false);
```

Comment:
I think it's just "Set up all questions as 'not completed' (false). When a question is completed, this value will be changed to true." or something.